tests/upload_pico_metrics.py

```diff
@@ -84,3 +84,4 @@
             upload_name = f"{name}_{i}" if many_results else name
             LOG.debug(f"Uploading metric: {upload_name} = {result}")
             metrics.put(upload_name, result)
+    metrics.publish()
```

Comment:
We should probably turn Metrics into a context manager to make it slightly less likely what one can forget to publish (what are the changes you want to add metrics but not publish them).

Reply:
https://github.com/jumaffre/cimetrics/issues/41